clientconn.go

```diff
@@ -725,10 +657,15 @@ func (cc *ClientConn) GetState() connectivity.State {
 // Notice: This API is EXPERIMENTAL and may be changed or removed in a later
 // release.
 func (cc *ClientConn) Connect() {
-	cc.exitIdleMode()
+	if err := cc.idlenessMgr.ExitIdleMode(); err != nil {
+		cc.addTraceEvent(fmt.Sprintf("error exiting idle mode: %v", err))
```

Comment:
This error message will contain repeated text since the error returned from the idleness manager already is:
```
return fmt.Errorf("channel failed to exit idle mode: %w", err)
```